aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js

```diff
@@ -0,0 +1,99 @@
+import React, { useContext, useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  CardFooter,
+  Grid,
+  Heading,
+  Page,
+  PageContent,
+  Paragraph,
+  ThemeContext,
+  ResponsiveContext,
+} from 'grommet';
+import { adjustPad } from '../../templates/Card/utils';
+import { activities } from '../card/data';
+
+const skeleton = { message: { start: 'Loading', end: 'Content Loaded' } };
+
+const columns = {
+  xsmall: ['auto'],
+  small: ['auto'],
+  medium: ['auto'],
+  large: ['auto', 'auto'],
+  xlarge: ['auto', 'auto', 'auto'],
+};
+
+export const DashBoardSkeletonExample = () => {
+  const theme = useContext(ThemeContext);
+  const breakpoint = useContext(ResponsiveContext);
+  const [loading, setLoading] = useState(true);
+  // needed for skeleton paragraph
+  const skeletonAlign = loading ? 'none' : 'start';
+  useEffect(() => {
+    setTimeout(() => setLoading(!loading), 3000);
+  }, [loading]);
+
+  const placeholders = [{}, {}, {}];
+  const items = loading ? placeholders : activities;
+
+  return (
+    // passing skeleton to Grid will skeletize all of the children that are
+    // text, button, box, paragraph, heading
+    <Page height="large">
+      <PageContent>
+        <Box skeleton={loading ? skeleton : undefined}>
+          <Grid columns={columns[breakpoint]} gap="medium">
+            {items.map((item, index) => {
+              console.log(item);
+              return (
+                <Card key={index}>
+                  <CardHeader
+                    direction="column"
+                    gap="none"
+                    align="start"
+                    pad={adjustPad('column', 'header', theme)}
+                  >
+                    {/* when using icon need to create a box to simulate skeleton behavior */}
+                    {!item.icon ? (
+                      <Box
+                        width="24px" // size of icon
+                        height="24px"
+                        background="background"
+                        flex="grow"
+                        margin={{ bottom: 'small' }}
+                      />
+                    ) : (
+                      <Box pad={{ bottom: 'small' }}>{item?.icon}</Box>
+                    )}
+                    <Heading
```

Comment:
<img width="817" alt="Screen Shot 2023-01-23 at 10 46 18 AM" src="https://user-images.githubusercontent.com/12522275/214123728-0fabfa98-1704-4b07-8e6c-9222161fe638.png">

it's a little strange how the padding on the right side of the card isn't there. I'd expect the heading to be a bit narrower so the padding can be there

Reply:
That's because the Heading skeleton has a width fixed by the theme. So as the cards shrink too much they start to not have space for the heading skeleton.

Reply:
Gotcha, wondering if we should adjust the width in theme to be something like:

```
width: { max: insert width, width: 100% }
```

I think that would allow us to set a max width and make sure the skeleton fills available space

Reply:
Heading with width 100% doesn't look good as a skeleton. It might be worth setting a better min/max in pixels though

Reply:
I can play around with this in Grommet and see for a min/max.